README.adoc

```diff
@@ -212,19 +276,80 @@ IMPORTANT: This command need to be executed from the same directory where the in
 
 NOTE: See more over it in the Readme of https://github.com/matryer/moq[moq]
 
-== Developing Helpers
-=== Errcheck
+* Using the mock
 
-The https://github.com/kisielk/errcheck[errcheck] is required to run the `make errcheck` command.
+In the created file you will see an implementation commented as the following example. This implementation will be used in the test file to mock the methods/func of this interface.
```

Comment:
`as in the following example` rather than `as the following example`

Reply:
Solved 👍 . Really tks.